http-clients/aws-crt-client/src/main/java/software/amazon/awssdk/http/crt/AwsCrtAsyncHttpClient.java

```diff
@@ -325,7 +325,7 @@ public interface Builder extends SdkAsyncHttpClient.Builder<AwsCrtAsyncHttpClien
          *
          * TODO: This is also used for the write buffer size. Should we rename it?
          */
-        Builder readBufferSizeInBytes(Integer readBufferSize);
+        Builder readBufferSizeInBytes(Long readBufferSize);
```

Comment:
Can we add documentation for this behavior? Something like: the max allowed read buffer size is Integer.MAX.
While we are at it, can we remove the TODO at line 326?

Reply:
Done.
On a similar note, what about the comment at line 69? Do we still want to keep it?

Reply:
Yeah, we can leave it as is now, and I'll create a PR to remove all of PREVIEW wordings when it's ready